en-US.js

```diff
@@ -394,6 +394,7 @@ export default {
       server_name_placeholder: 'Server name...',
       create_server: 'Create server',
       select_friends: 'Invite your friends to this server',
+      server_name_error: 'Server name must be at least 5 characters',
```

Comment:
maybe we can add a dot on the end 

the other similar errors also have a dot on the end

Reply:
@stavares843 Good idea, I must have forgotten!